WordPress/Resources/ar.lproj/InfoPlist.strings

```diff
@@ -1,5 +1,6 @@
-NSLocationUsageDescription = "WordPress would like to add your location to posts on sites where you have enabled geotagging.";
-NSLocationWhenInUseUsageDescription = "WordPress would like to add your location to posts on sites where you have enabled geotagging.";
-NSCameraUsageDescription = "لالتقاط صور أو مقاطع الفيديو لاستخدامها في مقالاتك.";
-NSPhotoLibraryUsageDescription = "لإضافة صور أو مقاطع فيديو إلى مقالاتك.";
-NSMicrophoneUsageDescription = "بالنسبة إلى مقاطع الفيديو المراد إضافة صوت إليها.";
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!--Warning: Auto-generated file, do not edit.-->
+<plist version="1.0">
+  <dict/>
+</plist>
```

Comment:
This and all the other locales apart from `it` are all empty, as expected because 418bcc18fe and a8fc965353 hack the process to bypass the GlotPress download and only introduce a few (three) new keys, which can all be seen in the `it` `.strings` file.

See also #18061:

> Note that the same files for any other locale you didn't add dummy keys to, and which thus won't have the corresponding keys in their *.lproj/Localizable.strings file to be extracted, it will result in empty XML plist files, replacing the ones that were present in the repo so far (and in OpenStep format) and had past translation.